securedrop/source_templates/index.html

```diff
@@ -38,7 +38,7 @@
           <div class="index-row">
             <div class="index-column index-left index-top">
               <h2 class="welcome-text">
-                {{ gettext('Submit documents for the first time') }}
+                {{ gettext('Submit for the first time') }}
```

Comment:
:+1:

Reply:
Please note that this text occurs in one of the Nagios monitoring checks of the source interface, so it will need to be updated there as well concurrent with this change being included in a release, or made more lenient. (H/T to @conorsch for flagging.)

Reply:
So if we modify the Nagios monitoring, the most robust modification is to monitor the metadata JSON endpoint on each source interface, which includes the version and submission key fingerprint used by the instance, see for example The Intercept (`http://intrcept32ncblef.onion/metadata`):

```
{"sd_version": "0.6", "gpg_fpr": "8AA5F557401AA06FF17656761EE602C06583211D"}
```